hw/ip/rv_plic/data/rv_plic.prj.hjson

```diff
@@ -4,9 +4,14 @@
 
 {
     name:               "rv_plic",
-    version:            "0.5",
-    life_stage:         "L1",
-    design_stage:       "D0",
-    verification_stage: "V0",
-    notes:              "will be verified at top level; formal at block level",
+    revisions: [
+      {
+        version:            "0.5",
+        life_stage:         "L1",
+        design_stage:       "D2",
+        verification_stage: "V2",
+        commit_id:          "6a3a7851b484e12185673887034f91d00e5dbbff",
```

Comment:
We're moving away from the idea of having a commit ID except for on
signoff. We figure the design is still fluid at this point until signoff, and
the commit history should be good enough. Thoughts @eunchan and
@sriyerg and @msfschaffner ?

Reply:
I think we should mandate that going from one revision to the next will require the commit id to be indicated (for the revision we updated from). For the new revision, no need to indicate the commit id if it is WIP.